src/plugins/expressions/public/_expression_renderer.scss

```diff
@@ -8,8 +8,13 @@
 }
 
 .expExpressionRenderer__expression {
+  display: flex;
+  flex-direction: column;
   width: 100%;
   height: 100%;
+  overflow: auto;
+  position: relative;
+  flex: 1 1 100%;
 }
```

Comment:
Will this scss change impact the style of UI components in other places?

Reply:
No they will not. These styles only affect `ReactExpressionRenderer` which is currently not used anywhere in the application.